src/ansi_color.go

```diff
@@ -92,6 +92,10 @@ func (a *AnsiColor) getAnsiFromColorString(colorString string, isBackground bool
 }
 
 func (a *AnsiColor) writeColoredText(background, foreground, text string) {
+	// Avoid emitting empty string with color codes 
```

Comment:
@lnu can you format this correctly?

Reply:
I'm fighting with golanci-lint on windows. I'll do this asap.